sdk/clientcore/core/pom.xml

```diff
@@ -69,6 +69,8 @@
     <!-- If JMH benchmarking was ran Checkstyle may fail on jmh_benchmark generated classes, ignore them. -->
     <checkstyle.excludes>**/generated/**/*.java,**/aalto/**/*.java,**/jackson/core/**/*.java</checkstyle.excludes>
 
+    <checkstyle.suppressionsLocation>checkstyle-suppressions.xml</checkstyle.suppressionsLocation>
```

Comment:
This should be the default setting in ClientCore parent and shouldn't be needed.